bee-protocol/src/worker/solidifier/mod.rs

```diff
@@ -9,10 +9,6 @@
 // an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
If there is only one file now, I'd not keep the folder and just do a file module in worker

Reply:
sure